conda_forge_tick/auto_tick.py

```diff
@@ -711,6 +715,10 @@ def _compute_time_per_migrator(mctx):
 
 
 def main(args: "CLIArgs") -> None:
+    # start profiler
+    pr = cProfile.Profile()
```

Comment:
you might consider a different variable name, `pr` is a bit overloaded at the moment.

Reply:
sorry for that

Reply:
done!